### ISSUE TITLE
cockatrice missing version in about dialog

### ISSUE THREAD

**Issue by ctrlaltca** [Homebrew/homebrew-games#575]
An user reported this bug on Cockatrice's issues: https://github.com/Cockatrice/Cockatrice/issues/2005
The version (composed by an hash and a date) is missing since it's extracted from the git commit.
I see two things that needs to be done here:
 1. fix cockatrice so that it can report the version even if it's not compiled from a git tree
 2. update the cockatrice formula

For the first point,  speaking as a cockatrice developer, we could add the ability of loading (forcing) the version string from a file, so that we can provide such a file in the stable tarballs, or the formula can insert such informations in the file when run.

If you have any better idea to solve the first point (maybe you already dealt with it for another project), i'll be glad to hear alternative possible solutions before implementing my idea.

**Comment by dunn** [comment]
Sorry for the delay in replying!

We could update the cockatrice formula to clone the repository instead of using the tarball.  Would a shallow clone work, if it just needs the current commit?  A deep clone would be a bit of a pain.

If you were to modify the build process so we could specify the version as a CMake flag or what have you, that would work too.

**Comment by ctrlaltca** [comment]
> Would a shallow clone work, if it just needs the current commit?

Sure, we are just running these two command using cmake in order to get the last commit hash and date:
```
> git log -1 --date=short "--pretty=%h"
3344083
> git log -1 --date=short "--pretty=%cd"
2016-07-01
```

**Comment by dunn** [comment]
Fixed in 5a83d428e3a23a1a2dfb079a1118d6a525be90a5; thanks for your patience!

**Comment by ctrlaltca** [comment]
Thank you for the fix 👍
